### PR TITLE
Extend clinvar form (fix #1340)

### LINKED ISSUE #1340: Clinvar fixes
- [x] Open clinvar submission in a blank page

- [x] OMIM field

- [x] HPO don't preselect, but checkboxes for the available terms

- [x] Remove : and stuff after in snp name (if present)

- [x] Zygosity in case data:
Do preselected but show all options regardless.
0/1 : if recessive (preselect compound hetero= otherwise hetero.
1/1 Homo, otherwise hemi if on sex chrom
Add something to focus on field

- [x] Testing laboratory -> Leave blank

- [x] Casedata stuff:
Clinical features is missing in the form
Comments on clinical significance and Comments on evidence should be there but they are not
Remove Evidence citation and comments on comments on clinical features

### PR DESCRIPTION
**Pull request by northwestwitch**
Improved the Clinvar submission form.

**Changes from the current master to be tested**
1. When clicking on the 'Submit to clinvar' button on a variant (must be a pinned variant) then the clinvar submission form opens on a new window.
1. Variant data form now contains both **HPO** and **OMIM** terms and the input becomes checkboxes instead of text fields. Example with on the left the old form and on the right the new one:

image
1. All options should be available for the zygosity in the casedata form
1. Clinical feature comment and Evidence citation fields are gone from the casedata form
1. "Comment on clinical significance" is now available in the variant data form.
1. Introduced a clinical features field in casedata form that behaves as in point 2 but has only selectable HPO terms

**How to test**:
1. Save a snv and a sv variant as clinvar submission objects, fill in all available fields and check that exported excel documents contain all the provided info.

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by dNil
- [x] tests executed by dNil

### CODE REVIEW

**Comment by northwestwitch** [comment]
I'm done with this one. Whoever has time!

**Comment by dnil** [comment]
Works just fine, as per test instructions!
Screenshot 2019-10-29 at 21 24 35
Screenshot 2019-10-29 at 21 23 51

File: scout/server/blueprints/variant/templates/variant/clinvar.html [hunk 1]
```diff
@@ -560,14 +581,21 @@ <h5>
                                   {% if sample.display_name == individual.display_name %}
                                     <strong>({{ sample.genotype_call }})</strong><br>
                                     <select name="zygosity@{{pinned._id}}" id="zygosity">
-                                      {% if sample.genotype_call == '1/1' %}
+                                      {% if sample.genotype_call == '1/1' and pinned.chromosome in ['X','Y'] %}
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/clinvar.html]: Isn't there a more elegant way to add that selected tag? Seems a little repetitive..

File: tests/parse/test_parse_clinvar_form.py [hunk 2]
```diff
@@ -3,28 +3,29 @@
 def get_submission_dict():
     #returns a clinvar submission dictionary with two variants and one case data to include. It's the format you obtain by converting a form request into a dict (form_dict = request.form.to_dict())
     test_form_fields_dict = {
-        'main_var': '3eecfca5efea445eec6c19a53299043b',
+        'main_var': 'test_snv_variant',
```
Comment (dnil) [inline tests/parse/test_parse_clinvar_form.py]: :+1:

File: scout/server/blueprints/variant/templates/variant/clinvar.html (hunk 1 above)
Comment (northwestwitch) [reply scout/server/blueprints/variant/templates/variant/clinvar.html]: There sure is, jinja style!

**Comment by northwestwitch** [comment]
Thanks for the review @dnil!

File: scout/server/blueprints/variant/templates/variant/clinvar.html [hunk 3]
```diff
@@ -581,22 +581,10 @@ <h5>
...
-                                        <option value="homozygote" selected>homozygote</option>
-                                        <option value="compound heterozygote">compound heterozygote</option>
-                                        <option value="single heterozygote">single heterozygote</option>
-                                      {% else %}
-                                        <option value="hemizygote">hemizygote</option>
-                                        <option value="homozygote">homozygote</option>
-                                        <option value="compound heterozygote" selected>compound heterozygote</option>
-                                        <option value="single heterozygote">single heterozygote</option>
-                                      {% endif %}
+                                      <option value="hemizygote" {%if sample.genotype_call == '1/1' and pinned.chromosome in ['X','Y'] %} selected {%endif%}>hemizygote</option>
```
Comment (dnil) [inline scout/server/blueprints/variant/templates/variant/clinvar.html]: 👍

**Pull request merged**